sk/static/animebytesmark.user.js

```diff
@@ -183,7 +184,12 @@ async function doAnimeBytes() {
 }
 
 function revealABEntries() {
-  document.head.insertAdjacentHTML('beforeend', '<style>a.pt-button.hidden {display: inline-flex !important;}</style>')
+  waitForKeyElements("a.pt-button", (elm) => {
+    const url = new URL(elm.dataset.href)
+    elm.href = 'https://animebytes.tv' + url.pathname + url.search
```

Comment:
this is risky as it can potentially, new URL('/path') is not valid, however HTMLAnchorElement.prototype.href *usually* includes the origin, I bet you there will be some firefox version that *doesnt* follow this standard, i think it would be safer to do
```suggestion
    elm.href = new URL(elm.dataset.href, 'https://animebytes.tv')
   ```